src/mtcp/mtcp_restart.c

```diff
@@ -1267,12 +1266,12 @@ remapMtcpRestartToReservedArea(RestoreInfo *rinfo)
   void *new_stack_start_addr = new_stack_end_addr - rinfo->old_stack_size;
 
   rinfo->new_stack_addr =
-    mtcp_sys_mmap(new_stack_start_addr,
-                  rinfo->old_stack_size,
-                  PROT_READ | PROT_WRITE,
-                  MAP_ANONYMOUS | MAP_PRIVATE | MAP_FIXED,
-                  -1,
-                  0);
+    mmap_fixed_noreplace(new_stack_start_addr,
+                         rinfo->old_stack_size,
+                         PROT_READ | PROT_WRITE,
```

Comment:
REPEATED:
The `MAP_FIXED` is redundant, since `map_fixed_noreplace()` will simply remove the `MAP_FIXED`.
We can keep it here, or remove `MAP_FIXED` with the idea that the name `map_fied_noreplace()` already says it all.
I'm open to either option (keeping it as is, or removing it).  Just a thought.